10371.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright 2020 Sierra Wireless. 
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+
+1. Redistributions of source code must retain the above copyright
+notice, this list of conditions and the following disclaimer.
+2. Redistributions in binary form must reproduce the above copyright
+notice, this list of conditions and the following disclaimer in the
+documentation and/or other materials provided with the distribution.
+3. Neither the name of the copyright holder nor the names of its
+contributors may be used to endorse or promote products derived
+from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+
+-->
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M.xsd">
+	<Object ObjectType="MODefinition">
+		<Name>DeviceExt</Name>
+		<Description1><![CDATA[This object extends Device(/3) Object. It mainly provides features allowing better monitor of Reboot and Reset Factory operation.]]></Description1>
```

Comment:
I use a short description here as it is adviced to use 134 character max.

(see https://github.com/OpenMobileAlliance/lwm2m-registry/wiki/Guidelines#description)

Reply:
If this "extends" the device object, should these be considered for a new version of the device object itself? If that's not good option, perhaps better to say "provides added functionality beyond.." or similar. And at large, maybe we (DMSE?) 
should think what kind of paradigm and design pattern we really want for this kind of extensions to the management objects.

Reply:
Definitely a new version of Device Object would be better.
This Object just aims to fix that it seems to me as an LWM2M core issue that we discuss in https://github.com/OpenMobileAlliance/OMA_LwM2M_for_Developers/issues/280#issuecomment-363483045

But we need this new version for LWM2M 1.0.x and I don't know if this is doable ? 
If this is possible then we don't really need this new object.